### ISSUE TITLE
Bluebook style?

### ISSUE THREAD

**Issue by rmzelle** [zotero/styles-repo#14]
What was the deal with https://github.com/zotero/styles-repo/blob/master/include/bluebook-19th.csl again? I can't even find this version in the repo right now: https://www.zotero.org/styles?q=bluebook. I recall Frank asked to have it removed from the central CSL VCS at some point. @fbennett? @adam3smith?

**Comment by fbennett** [comment]
Yeah, I raised a flag about BB out of am excess of sensitivity, but I think the consensus was that it was okay to stay. IIRC though that particular version was not of very good quality.

**Comment by adam3smith** [comment]
so what I'd like to do, mainly as a political act, is to go ahead and code
a high quality version of Baby Blue. We could just put that out instead.
Or, if we want to be cheeky, make Bluebook a dependent style of it. :P

On Tue, Feb 23, 2016 at 7:51 AM, Frank Bennett <notifications@github.com>
wrote:

> Yeah, I raised a flag about BB out of am excess of sensitivity, but I
> think the consensus was that it was okay to stay. IIRC though that
> particular version was not of very good quality.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zotero/styles-repo/issues/14#issuecomment-187686652>.
>



-- 
Sebastian Karcher, PhD
www.sebastiankarcher.com

**Comment by fbennett** [comment]
Juris-M would be a better fit, for the extended fields and item types, and to handle jurisdiction variants.

**Comment by adam3smith** [comment]
yeah, but also a more significant time investment and visible to a smaller
audience, so there's a trade-off.

On Tue, Feb 23, 2016 at 8:57 AM, Frank Bennett <notifications@github.com>
wrote:

> Juris-M would be a better fit, for the extended fields and item types, and
> to handle jurisdiction variants.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zotero/styles-repo/issues/14#issuecomment-187709295>.
>



-- 
Sebastian Karcher, PhD
www.sebastiankarcher.com

**Comment by fbennett** [comment]
It should work okay if you limit cases to the US Supreme Court, and statutes to the US Code.

**Comment by dstillman** [comment]
> so what I'd like to do, mainly as a political act, is to go ahead and code a high quality version of Baby Blue

That'd be great. We could even put "(US Code)" or something into the title to indicate its limitations?

**Comment by fbennett** [comment]
Well, I was exaggerating just a bit there ... actually, if the style front matter could include a pointer to Juris-M, it would be very helpful. JM no longer modifies the Zotero DB, so users would not be trapped if they decide to go back.

**Comment by adam3smith** [comment]
I'll certainly keep pointing to juris-m at every opportunity (including in
the style summary, sure) and as you know it will remain the only way to go
for full legal support for certainly the near future. And I'd be happy to
collaborate on a juris-m version -- but whereas I can just grab a bottle of
wine and do this in vanilla CSL in an evening, csl-m would take me more
like a weekend...

On Tue, Feb 23, 2016 at 5:14 PM, Frank Bennett <notifications@github.com>
wrote:

> Well, I was exaggerating just a bit there ... actually, if the style front
> matter could include a pointer to Juris-M, it would be very helpful. JM no
> longer modifies the Zotero DB, so users would not be trapped if they decide
> to go back.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zotero/styles-repo/issues/14#issuecomment-187940168>.
>



-- 
Sebastian Karcher, PhD
www.sebastiankarcher.com
